R/test_de.R

```diff
@@ -243,6 +250,18 @@ test_de_q <- function(fit,
       diag(ridge_penalty, nrow = length(ridge_penalty)) %*% rot
     }
     lfc <- fit$Beta %*% cntrst / log(2)
+
+    if (compute_lfc_se) {
+      pred <- predict(fit, se.fit=TRUE, type='link',
+                      offset=0, #for asserting equality with lfc
+                      newdata=matrix(cntrst, nrow=1))
+
+      lfc_se <- pred$se.fit[,1] / log(2)
+
+      stopifnot(rownames(pred$se.fit) == rownames(fit$Beta))
+      stopifnot(all.equal(pred$fit / log(2), lfc))
```

Comment:
Added these 2 assertions out of excessive caution, but it might be overkill...should we remove them?